server/activityItem/model.ts

```diff
@@ -1,27 +1,28 @@
-export default (sequelize, dataTypes) => {
-	return sequelize.define(
-		'ActivityItem',
-		{
-			id: sequelize.idType,
-			kind: { type: dataTypes.TEXT, allowNull: false },
-			pubId: { type: dataTypes.UUID },
-			payload: { type: dataTypes.JSONB },
-			timestamp: {
-				type: dataTypes.DATE,
-				defaultValue: sequelize.Sequelize.NOW,
-				allowNull: false,
-			},
-			communityId: { type: dataTypes.UUID, allowNull: false },
-			actorId: { type: dataTypes.UUID },
-			collectionId: { type: dataTypes.UUID },
+import { DataTypes as dataTypes, NOW } from 'sequelize';
+import { sequelize } from '../sequelize';
+
+export const ActivityItem = sequelize.define(
+	'ActivityItem',
+	{
+		id: sequelize.idType,
+		kind: { type: dataTypes.TEXT, allowNull: false },
+		pubId: { type: dataTypes.UUID },
+		payload: { type: dataTypes.JSONB },
+		timestamp: {
+			type: dataTypes.DATE,
+			defaultValue: NOW,
+			allowNull: false,
 		},
-		{
-			indexes: [
-				{ fields: ['communityId'], method: 'BTREE' },
```

Comment:
this is the one change to the `define`s. `method` is not a valid property, i don't think it ever was checking the sequelize v4 and v5 docs. But I think `BTREE` is the default index method anywan